Interop/ModelCard/src/main/java/ModelCard.java

```diff
@@ -0,0 +1,74 @@
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.tribuo.Model;
+import org.tribuo.evaluation.Evaluation;
+import org.tribuo.interop.ExternalModel;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+public class ModelCard {
```

Comment:
For the time being, let's make all the classes in here `final`. I don't want to have to think about subclasses when we evolve this to have feature metrics or correlations in it.

Reply:
Done as suggested.